d2l-typography-shared-styles.html

```diff
@@ -26,6 +26,14 @@
 					text-transform: uppercase;
 				};
 
+				--d2l-body-compact-text: {
+					color: var(--d2l-color-ferrite);
+					font-family: inherit;
+					font-size: 0.8rem;
+					font-weight: 400;
+					line-height: 1rem;
```

Comment:
My manual testing also hasn't turned up any problems with compact having a `line-height` of `1rem`. I'd like to leave it like this for now, since it matches the design guidance. This class isn't used anywhere currently, so if there are problems that crop up during usage, we can address this setting then.

The issues we'd seen before had to do with setting `line-height` to `100%`, so I don't think we're going to have as much of a pitfall here.